.postcssrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  map: true,
+  plugins: {
+    'postcss-discard-duplicates': {},
+    autoprefixer: {},
+    'css-mqpacker': {},
+    'postcss-banner': {
+      banner: 'Copyright 2020 Workday, Inc.',
```

Comment:
I can't remember the exact rules for writing copyright notices but do we want to keep the year we marked it copyrighted? Or do we want to keep the year synchronized or even like a date range (e.g. `2020 - 2021` for next year).

Anyway, it's a nit, but adding a programmatic year probably wouldn't hurt